### ISSUE TITLE
Check why parent::func() is not resolved

### ISSUE THREAD

**Issue by zobo** [zobo/php-language-server#27]
Why is LSP not offering definition position of `parent::b()`?

```
class B {
    function b() {

    }
}

class A extends B {
    function a () {
        $a = parent::b();
    }
}
```

**Comment by zobo** [comment]
The problem is that functions get stored in the index either as `Namespace\Class->func()` or `Namespace\Class::func()` depending on if they are not static or static defined.
When making a `parent::func()` call the parses can't figure out if it's looking for a static method or a non-static method.

Maybe try both?

**Comment by daslicht** [comment]
i just tried intelliphnese and there it works at lest withon clases, just cmd click on a function

**Comment by zobo** [comment]
Fixed in #29 will be released soon.

**Comment by daslicht** [comment]
intelephense.com premium got this feature:

https://intelephense.com
